### ISSUE TITLE
Width of 100% not working in modal

### ISSUE THREAD

**Issue by awilson69** [davidstutz/bootstrap-multiselect#1118]
I have several places where I'm using this control on standard pages and it's rendering just fine, but I tried adding it to a modal window and it's ignoring the width of 100% that I've set.  

If I use specific px values, then the width adjusts as expected, but that's not really an option here.

Any ideas?  I've been playing around with the CSS most of the day and I've not had any luck so far.

![BS-MultiSelect-Issue](https://user-images.githubusercontent.com/27510963/67041985-d8b5f280-f0f4-11e9-8726-0b1fe4a8de6c.png)

**Comment by tiesont** [comment]
I seem to recall this being a timing issue. Basically, the element doesn't have a width until the modal is shown, which throws off the drop-down calculations. Been a while since I've actively used this plugin, though, so that's mostly a guess at this point.

**Comment by awilson69** [comment]
Thank you, tiesont.  I tried playing around with timing (destroying, recreating, etc.) and didn't get anywhere, but once I removed the html structure around this control and added things back one at a time, I found the issue.  

For anyone else who encounters this...

I had the multi-select control wrapped inside a div with a class of "input-group'.  When I removed that, the control renders it's width perfectly.
